main.py

```diff
@@ -148,44 +148,27 @@ def request_linkedin_data() -> List[object]:
         "div",
         class_="base-card relative w-full hover:no-underline focus:no-underline base-card--link base-search-card base-search-card--link job-search-card",
```

Comment:
Is it possible to narrow this down to avoid issues down the line where classes change?

Reply:
Sure!